lib/rubocop/cop/rspec/factory_bot/consistent_parentheses_style.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module FactoryBot
+        # Use a consistent style for parentheses in factory bot calls
+        #
+        # @example 
+        #
+        #   # bad
+        #
+        #   create :user
+        #   build(:user)
+        #   create(:login)
+        #   create :login
+        #
+        # @example `EnforcedStyle: require_parentheses`
+        #
+        #   # good
+        #   create(:user)
+        #   create(:user)
+        #   create(:login)
+        #   build(:login)
+        #
+        # @example `EnforcedStyle: omit_parentheses`
+        #
+        #   # good
+        #   create :user
+        #   build :user
+        #   create :login
+        #   create :login
+        #
+        class ConsistentParenthesesStyle < Base
+          extend AutoCorrector
+          include ConfigurableEnforcedStyle
+
+          def self.autocorrect_incompatible_with
+            [Style::MethodCallWithArgsParentheses]
+          end
+
+          MSG_ENFORCE_PARENS = 'Prefer method call with parentheses'
+          MSG_OMIT_PARENS = 'Prefer method call without parentheses'
+
+          FACTORY_CALLS = Set[:create, :build, :create_list,
+                              :build_list, :create_pair,
+                              :build_stubbed_list, :atributes_for,
+                              :build_stubbed, :build_pair ].freeze
```

Comment:
Where does this list of method names come from? I found a list of method names at https://www.rubydoc.info/gems/factory_bot/FactoryBot/Syntax/Methods, but that list also includes `attributes_for_list`, `attributes_for_pair` and `build_stubbed_pair` – so I wonder if those are deliberately omitted.

There’s a typo in `:atributes_for`.

Reply:
Thank you very much ^^ The 3 missing were not left out on purpose. I guess I forgot them when the pr was created.